
package com.barryku.cloud.sforce.partner;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.4.2
 * 2011-09-11T16:54:21.633-07:00
 * Generated source version: 2.4.2
 */

@WebFault(name = "InvalidNewPasswordFault", targetNamespace = "urn:fault.partner.soap.sforce.com")
public class InvalidNewPasswordFault_Exception extends Exception {
    
    private com.barryku.cloud.sforce.partner.InvalidNewPasswordFault invalidNewPasswordFault;

    public InvalidNewPasswordFault_Exception() {
        super();
    }
    
    public InvalidNewPasswordFault_Exception(String message) {
        super(message);
    }
    
    public InvalidNewPasswordFault_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public InvalidNewPasswordFault_Exception(String message, com.barryku.cloud.sforce.partner.InvalidNewPasswordFault invalidNewPasswordFault) {
        super(message);
        this.invalidNewPasswordFault = invalidNewPasswordFault;
    }

    public InvalidNewPasswordFault_Exception(String message, com.barryku.cloud.sforce.partner.InvalidNewPasswordFault invalidNewPasswordFault, Throwable cause) {
        super(message, cause);
        this.invalidNewPasswordFault = invalidNewPasswordFault;
    }

    public com.barryku.cloud.sforce.partner.InvalidNewPasswordFault getFaultInfo() {
        return this.invalidNewPasswordFault;
    }
}
